### DISCUSSION TITLE
Max and min stake working properly?

### DISCUSSION THREAD

**Discussion by jbn1981** [cloudbet/docs#85] [Q&A]
Hi,

I can see differences between max/min stake shown at the website and the API.

For example, this event:

![cloudbet-stake](https://github.com/cloudbet/docs/assets/32179115/7b11bfd7-4651-4ca5-a503-fc1ef3f54395)

![image](https://github.com/cloudbet/docs/assets/32179115/38815565-a1cc-43fd-a207-4818dd4dd94c)

Look:

**WEBSITE**
Max Stake: 167493
Min Stake: 10

**API**
Max Stake: 98.00686
Min Stake: 0.1

Is that a bug or am I missing something?

Thanks.

**Comment by kgravenreuth** [comment]
Hi @jbn1981 

The min/max stake values on the Cloudbet website are displayed in your chosen currency whereas in the Cloudbet API, they are displayed in Euros currently. We are considering an improvement around this in https://github.com/cloudbet/docs/discussions/82 while we can't give you an expected timeline at the moment. 

Furthermore, the min stake values could be different based on our internal decisions.

Could you compare the min/max stake between the Cloudbet website and the Cloudbet API again for your account while taking the currency differences into account? Please let us know if you have further questions. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by jbn1981** [comment]
Yeah, now it makes sense.

Thanks.
